scripts/SHI/scpSHI_Jetz_complete.yml

```diff
@@ -0,0 +1,93 @@
+script: scpSHI_Jetz_complete.R
+description: 
+  This script loads the data needed to calculate the area and connectivity scores 
+  to calculate the Species Habitat Index (SHI) using Global Forest Watch data to measure 
+  changes in the habitat of the species
+inputs:
+  species:
+    label: species
+    description: Scientific name of the species
+    type: text
+    example: Saguinus oedipus
+  ref_system:
+    label: Projection
+    description: Reference system for the area of interest
+    type: int
+    example: 3116
+  elev_margin:
+    label: Elevation buffer
+    description: Magnitude of buffer to add to the elevation ranges of the species (usualy in meters), if no buffer then 0
+    type: int
+    example: 0
+  forest_threshold:
+    label: Tree cover percentage 
+    description: 
+      Percentage of tree cover of reference for the Global Forest Watch data according to the tolerance of the species. 
+      This value is given according to suitable habitat of the species. If not available, use Map of Life Values 
+      (e.g. [https://mol.org/species/range/Saguinus_oedipus]) 
```

Comment:
Percentage of tree cover suitable for species based on tolerance of species values from Global Forest Watch  (if not available, use Map of .........).

Reply:
Changed to:
"Percentage of tree cover required by the species, that will be used to filter the Global Forest Watch data. 
      This value is given according to suitable habitat of the species. If not available, use Map of Life Values 
      (e.g. [https://mol.org/species/range/Saguinus_oedipus]) "